index.html

```diff
@@ -55,9 +56,9 @@ <h2>Post 2</h2>
     </div>
   </section>
   <footer>
-    <p>&copy; 2017 Skilly</p>
+    <p>&copy; 2018 Skilly</p>
```

Comment:
`2019` would be more appropriate, however I can't suggest any more changes since you've deleted your branch.